src/types/instruction.rs

```diff
@@ -78,3 +81,38 @@ impl Instruction {
         }
     }
 }
+
+// Returns True if the given instruction looks like a call instruction.
+pub(crate) fn is_call_instruction(encoded_instruction: &BigInt, imm: Option<BigInt>) -> bool {
+    let encoded_i64_instruction: i64 = match encoded_instruction.to_i64() {
+        Some(num) => num,
+        None => return false,
+    };
```

Comment:
I think the following should work (tho not an important change, it's neat information):
```suggestion
    let encoded_i64_instruction: i64 = encoded_instruction.to_i64()
        .ok_or(false);
```

Reply:
That's because `Result<T, U>` can have any type `U` as its other variant.

Reply:
Couldn't get it to work

Reply:
Because I was wrong and didn't notice the types mismatch :facepalm: